README.md

```diff
@@ -42,7 +42,13 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
 ```js
-// write code here
+// var food = ["apple", "banana","strawberry"];
+function eat(food) {
+  console.log("I want to eat" + food);
+}
+for each(var i = 0;i <fruit.length; i++){
+  eat(food)
+}
```

Comment:
This would throw an error because `fruit` isn't defined, and `each` isn't a method. [Check the solution.](https://github.com/ga-wdi-quizzes/week-02/blob/solution/README.md#question-3)